Sources/MapboxSearchUI/Resources/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>NSPrivacyAccessedAPITypes</key>
```

Comment:
I think this is safe because the UI module imports the MapboxSearch data module, and the data module has its own xcprivacy. Similar to how MapboxCommon has its own xcprivacy.